### PR TITLE
Add FairMOT and Deep Sort ROS nodes

### PR DESCRIPTION
**Pull request by iliiliiliili**
Added FairMOT and Deep Sort 2D Tracking ROS nodes.

### CODE REVIEW

File: projects/opendr_ws/src/perception/README.md [hunk 1]
```diff
@@ -9,12 +9,19 @@ Dataset nodes take a `DatasetIterator` object that shoud returns a `(Data, Targe
 If the type of the `Data` object is correct, the node will transform it into a corresponding ROS message object and publish it to a desired topic.
 
 ### Point Cloud Dataset ROS Node
-To get a point cloud from a dataset on the disk, you can start a `point_cloud_dataset.py` node as:
+To get an image from a dataset on the disk, you can start a `point_cloud_dataset.py` node as:
```
Comment (passalis) [inline projects/opendr_ws/src/perception/README.md:12]: An image or an point cloud? Are we sure about this change?

File: src/opendr/perception/object_tracking_2d/fair_mot/object_tracking_2d_fair_mot_learner.py [hunk 2]
```diff
@@ -375,7 +375,7 @@ def infer(self, batch, frame_ids=None, img_size=(1088, 608)):
 
         for image, frame_id in zip(batch, frame_ids):
 
-            img0 = image.numpy().transpose(1, 2, 0)[..., ::-1]  # BGR
+            img0 = image.convert("channels_first").transpose(1, 2, 0)[..., ::-1]  # BGR
```
Comment (passalis) [inline src/opendr/perception/object_tracking_2d/fair_mot/object_tracking_2d_fair_mot_learner.py:378]: Why not `convert("channels_last", "bgr")`?

**Review by passalis — COMMENTED** [review]
Thank you! Overall, seems fine. I have two minor comments.

**Review by passalis — APPROVED** [review]
Thank you!

**Review by ad-daniel — APPROVED** [review]

**Pull request merged**